### PR TITLE
Database monk's start now with jab in actionbar

### PR DESCRIPTION
**Pull request by zTerragor**
working. port from sf

### CODE REVIEW

File: sql/updates/master/world/2023_11_06_23_18.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+(2, 10, 9, 20572, 0), -- Blood Fury (Racial) 
+(3, 10, 9, 20594, 0), -- Stoneform (Racial)
+(4, 10, 9, 58984, 0), -- Shadowmeld (Racial)
+(5, 10, 9, 20577, 0), -- Cannibalize (Racial)
+(6, 10, 9, 20549, 0), -- War Stomp (Racial)
+(7, 10, 9, 20589, 0), -- Escape Artist (Racial)
+(10, 10, 9, 28730, 0), -- Arcane Torrent (Racial)
+(11, 10, 9, 59542, 0), -- Gift of the Naaru (Racial)
+(24, 10, 9, 107079, 0), -- Quaking Palm (Racial)
+(1, 10, 81, 59752, 0), -- Every Man for Himself (Racial)
```
Comment (leelf00) [inline sql/updates/master/world/2023_11_06_23_18.sql:14]: from here doubled inserts

File: sql/updates/master/world/2023_11_06_world_01.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+(2, 10, 0, 100780, 0),
+(3, 10, 0, 100780, 0),
+(4, 10, 0, 100780, 0),
+(5, 10, 0, 100780, 0),
+(6, 10, 0, 100780, 0),
+(7, 10, 0, 100780, 0),
+(10, 10, 0, 100780, 0),
+(11, 10, 0, 100780, 0),
+(24, 10, 0, 100780, 0),
+(1, 10, 72, 100780, 0),
```
Comment (leelf00) [inline sql/updates/master/world/2023_11_06_world_01.sql:17]: also double insert

File: sql/updates/master/world/2023_11_06_world_01.sql [hunk 3]
```diff
@@ -0,0 +1,26 @@
+DELETE FROM `playercreateinfo_spell` WHERE `Spell`=100780;
```
Comment (leelf00) [inline sql/updates/master/world/2023_11_06_world_01.sql:1]: Our DB don't have this table.Maybe missing in core

Comment (zTerragor) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]: https://github.com/ProjectSkyfire/SkyFire_548/pull/87/commits/080f6230d8a5469fcd7b1caeedb04769aee06a53

Comment (zTerragor) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]: [src/server/game/Globals/ObjectMgr.cpp](https://github.com/ProjectSkyfire/SkyFire_one/blob/7b3f0e801a2769933fbb67af9164ddacd330fbdc/src/server/game/Globals/ObjectMgr.cpp#L2121)

Comment (zTerragor) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]: <html>
<body>

{
--
2119 | QueryResult_AutoPtr result = QueryResult_AutoPtr(NULL);
2120 | if (sWorld->getConfig(CONFIG_START_ALL_SPELLS))
2121 | result = WorldDatabase.Query("SELECT race, class, Spell, Active FROM playercreateinfo_spell_custom");
2122 | else
2123 | result = WorldDatabase.Query("SELECT race, class, Spell, Active FROM playercreateinfo_spell");
2124 |  
 
2142 | sLog->outErrorDb("Wrong race %u in playercreateinfo_spell table, ignoring.", current_race);
 
2149 | sLog->outErrorDb("Wrong class %u in playercreateinfo_spell table, ignoring.", current_class);


</body>
</html>

Comment (leelf00) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]: Ok,I will add playercreateinfo_spell system to core.

Comment (zTerragor) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]:
```

2119 | QueryResult_AutoPtr result = QueryResult_AutoPtr(NULL);
2120 | if (sWorld->getConfig(CONFIG_START_ALL_SPELLS))
2121 | result = WorldDatabase.Query("SELECT race, class, Spell, Active FROM playercreateinfo_spell_custom");
2122 | else
2123 | result = WorldDatabase.Query("SELECT race, class, Spell, Active FROM playercreateinfo_spell");
2124 |  
```
 
2142 | sLog->outErrorDb("Wrong race %u in playercreateinfo_spell table, ignoring.", current_race);
 
2149 | sLog->outErrorDb("Wrong class %u in playercreateinfo_spell table, ignoring.", current_class);


</body>
</html>

Comment (zTerragor) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]: /*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
--
12 | /*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
13 |  
14 | -- Dumping structure for table sf_master.playercreateinfo_spell
15 | DROP TABLE IF EXISTS `playercreateinfo_spell`;
16 | CREATE TABLE IF NOT EXISTS `playercreateinfo_spell` (
17 | `racemask` int unsigned NOT NULL DEFAULT '0',
 
24 | -- Dumping data for table sf_master.playercreateinfo_spell: 261 rows
25 | /*!40000 ALTER TABLE `playercreateinfo_spell` DISABLE KEYS */;
26 | INSERT INTO `playercreateinfo_spell` (`racemask`, `classmask`, `Spell`, `Note`) VALUES
 
288 | /*!40000 ALTER TABLE `playercreateinfo_spell` ENABLE KEYS */;

Comment (leelf00) [reply sql/updates/master/world/2023_11_06_world_01.sql:1]: playercreateinfo_spell  maybe replace with playercreateinfo_skill

**Comment by leelf00** [comment]
Fixed by https://github.com/Legends-of-Azeroth/Legends-of-Azeroth-Pandaria-5.4.8/pull/268

**Pull request closed without merging**